### DISCUSSION TITLE
want to create first pull request

### DISCUSSION THREAD

**Discussion by ayush20021** [simple-icons/simple-icons#5791] [General]
Hii.  guys I am a beginner in web development and want to contribute in opensource and want to solve real world problems.
So can you guys help me out.
I can do some basic short of tasks

**Comment by adamrusted** [comment]
Hey @ayush20021 - as discussed previously, there's a few [good first issues](https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22) that you can take a look at. These are usually icons that're easily converted, or already in monochrome as needed by our project.

Following our [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon), and using your preferred vector editing software make the icon fit onto a 24*24px canvas (maintaining the correct aspect ratio).

The data in `_data/simple-icons.json` can be edited in any text editor, and you can glean the formatting from the guidelines above, or by looking at other icons already in the collection.

If you're stuck on how to use GitHub etc, there's plenty of resources like [this](https://lab.github.com/) that'll teach you what is possible with git. Again, our contribution guidelines contain detailed steps on forking, cloning and editing the project on your local machine. You'll need Node.js installed, and some form of Git client (GitHub Desktop / Git CLI) for things to run smoothly. Any other issues, feel free to ask them here - or in the issue you're assigned to.

**Comment by ayush20021** [comment]
I am trying

**Reply by fanatiicx** [reply]
@ayush20021 I found the [First Timers Only](https://www.firsttimersonly.com/) website useful before digging in further into the contribution guidelines.
